tools/genomespace/genomespace_exporter.xml

```diff
@@ -1,54 +1,42 @@
 <?xml version="1.0"?>
-<tool name="GenomeSpace Exporter" id="genomespace_exporter" require_login="True" version="0.0.4">
+<tool name="GenomeSpace Exporter" id="genomespace_exporter" version="0.0.4">
     <description> - send data to GenomeSpace</description>
-    <command interpreter="python">genomespace_exporter.py 
-        --genomespace_site "prod"
-        #assert $__user__, Exception( 'You must be logged in to use this tool.' )
-        #set $username = $__user__.preferences.get( 'genomespace_username', None )
-        #set $token = $__user__.preferences.get( 'genomespace_token', None )
-        #assert None not in ( $username, $token ), Exception( 'You must associate a GenomeSpace OpenID with your account and log in with it.' )
+    <command interpreter="python">genomespace_exporter.py
+        #set $target_folder = $genomespace_browser.split("^")[0]
+        #set $token = $genomespace_browser.split("^")[1] or $__user__.preferences.get( 'genomespace_token', None )
```

Comment:
I think the environment variable would be a great fit. The only sensitive piece of information here is the token, and if we can pass that off as an env var, that should be fairly safe. I'll test that now.